### ISSUE TITLE
Unable to get playSegments working

### ISSUE THREAD

**Issue by lukeoly** [tanisha-verma/lottie-web-react#1]
I'm having trouble getting the playSegments to work and then update the segment using the event listener. 

I'm not sure if my formatting is the issue or it's something else. Do you have an example of how this is meant to be written?

Here's my current implementation:

```
import thoughtBubbles from "../../../assets/animations/thoughtBubbles";

const [bubble, setBubble] = useState([0,24]);
const [loop, setLoop] = useState(false);
const [playingState, setPlayingState] = useState('play')

<Lottie
    options={{
        loop:  loop,
        autoplay:  true,
        animationData:  thoughtBubbles
    }}
    playSegments={{
        segments:bubble,
        forceFlag: true
    }}
    playingState={playingState}
    id="thought_bubble"
    className={styles["thought-bubble"]}
    eventListeners={[
        {eventName: 'complete', callback: () => {
            setLoop(true);
            setBubble([25, 250]);
            setPlayingState("play");
            console.log("loop", loop);
            console.log("bubble", bubble);
        }}
    ]}
/>
```
In this example, I'm trying to play a segment and then update the segment on complete.
If the `forceFlag` is set to `false`, the segment seems to be ignored and the entire animation plays out. If I set the `forceFlag` to `true`, it doesn't play at all.
Am I implementing this incorrectly?

I also noticed that if an id isn't specified on the `<Lottie>` component, it adds `id="lottie-web-react"`. This could be a problem if using multiple animations on the one page.

**Comment by lc-tanishaverma** [comment]
There was an issue with the code , I have fixed it in version v1.0.3 and added an example usage of segments.

**Comment by lukeoly** [comment]
Thanks @lc-tanishaverma. 
I can confirm this is working. I did however notice that the segments don't work if the force flag is not present or if it is set to false
